includes/admin/import/class-batch-import-downloads.php

```diff
@@ -106,6 +106,17 @@ public function process_step() {
 					}
 				}
 
+				if ( empty( $args['post_author'] ) ) {
+					$user = wp_get_current_user();
+					$args['post_author'] = $user->user_login;
+				} else {
+					$user = get_user_by( 'user_login', $args['post_author'] );
```

Comment:
@pippinsplugins yeah that's an improvement. Originally our code simply worked with `user_login`, not any other fields, I was going for minimal change set.